flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/scala/expressionDsl.scala

```diff
@@ -564,6 +564,13 @@ trait ImplicitExpressionOperations {
   def overlay(newString: Expression, starting: Expression, length: Expression) =
     Overlay(expr, newString, starting, length)
 
+  /**
+    * Returns a string resulting from replacing all substrings
```

Comment:
Returns a string with all substrings that match the regular expression being replaced.